multi-server-install-planning.adoc

```diff
@@ -187,12 +187,14 @@ components with the following default configuration.
 
 [NOTE]
 The following ports are used either by Zimbra Proxy or by Zimbra Mailbox
-(if Proxy is not configured). + 
+(if Proxy is not configured). +
 **If you have any other services running on these ports, turn them off.**
 
 End clients connect directly to Zimbra Proxy, using the Zimbra Proxy
 Ports. Zimbra Proxy connects to the Route Lookup Handler/NLE (which
 resides on Zimbra Mailbox server) using the Zimbra Mailbox Ports.
+The proxy also optionally connects to the IMAPD service on the Zimbra IMAPD ports
+if IMAPD is installed and configured at the global level or for a specific set of mailbox servers.
```

Comment:
nit: perhaps change "for" with "on"?
Also, shouldn't "Zimbra" be replaced with some {variable}?